lib/discourse_code_review/github_repo.rb

```diff
@@ -165,6 +165,7 @@ def followees(ref)
       result = []
 
       git_repo.commit(ref).message.lines.each do |line|
+        next if line =~ /^revert\b/i
```

Comment:
Sorry to quibble again, but could we check just the first line? You should be able to change `each` to `each_with_index` to get the line number.